### DISCUSSION TITLE
Override translation / translation key

### DISCUSSION THREAD

**Discussion by ivanleskovar** [orchidsoftware/platform#2678] [Q&A] [answered]
Hi, 

is there any way to override for example text in table footer: "Configure columns".

**Comment by c-gross** [comment, marked as answer]
You can use the translation json to change it for a language.
Otherwise you need to override the template file: vendor/orchid/platform/resources/views/layouts/pagination.blade.php
But then you have to take care of changes after orchid updates.

**Reply by ivanleskovar** [reply]
Thx, 
can you give mi more info how to do "You can use the translation json to change it for a language."?

**Reply by c-gross** [reply]
It's the default laravel translation
https://laravel.com/docs/10.x/localization
